### PR TITLE
[TRADE-22] Remove Order FeeCap

### PR DESCRIPTION
**Pull request by haoze-grvt**
- Remove fee cap fields from Order struct and signatures 
 - Use hard-coded fee cap of 20 bps instead

### CODE REVIEW

File: contracts/exchange/api/TradeContract.sol [hunk 1]
```diff
@@ -305,8 +305,8 @@ abstract contract TradeContract is ConfigContract, FundingAndSettlement, RiskChe
       }
     }
 
-    // Check that the fee paid is within the cap
-    int32 feeCapRate = isMakerOrder ? order.makerFeePercentageCap : order.takerFeePercentageCap;
+    // Check that the fee paid is within the cap of 20 bps
+    int32 feeCapRate = 2000;
```
Comment (minhbsq) [inline contracts/exchange/api/TradeContract.sol:309]: move this to a constant instead?

**Review by Aaronong — APPROVED** [review]
LGTM from a correctness perspective.
Confused at merging into `spearbit-audit-fixes`.
We are doing a hard reset on `dev` now. 
How will we pull this in?

**Comment by haoze-grvt** [comment]
> LGTM from a correctness perspective. Confused at merging into `spearbit-audit-fixes`. We are doing a hard reset on `dev` now. How will we pull this in?

We merge spearbit fixes into `main` in `platform` and `spearbit-audit-fixes` in this repo(at auditor request) => the current contract main is not compatible with `platform` `main` because of audit fixes. We will start enforcing risk-chain parity again once we merge all fixes tomorrow.

**Comment by Aaronong** [comment]
Just a quick confirmation.
From what I'm hearing, I can break `risk <> chain` anyway I like right now. 
As long as I put in RTF tests, and inform you. 
Re-deployment wise. We can move on with FE + BE, without worrying about whether chain is broken.

**Comment by haoze-grvt** [comment]
> Just a quick confirmation. From what I'm hearing, I can break `risk <> chain` anyway I like right now. As long as I put in RTF tests, and inform you. Re-deployment wise. We can move on with FE + BE, without worrying about whether chain is broken.

Yes, the contract doesn't have to work if we don't need to test withdrawal. Free money will still work as it doesn't depend on the `contract`, only the chain.

**Pull request merged**